lib/service/invoices.go

```diff
@@ -157,13 +157,14 @@ func createLnRpcSendRequest(invoice *models.Invoice) (*lnrpc.SendRequest, error)
 	if err != nil {
 		return nil, err
 	}
+	invoice.DestinationCustomRecords[KEYSEND_CUSTOM_RECORD] = preImage
```

Comment:
is this now always set? not only on keysend payements?

Reply:
On line 143, if the invoice is not keysend, we have an early return.

Reply:
ah, I missed that. ok, thanks.

in such code branches I prefer using if/else to make it clear

Reply:
I prefer the early return, as described in the [Zen of Go](https://the-zen-of-go.netlify.app/).

Reply:
:+1:
(just might be more easy to miss, but 🐐 )